generators/template/codegen/package.json

```diff
@@ -30,9 +30,7 @@
     "@fern-api/fs-utils": "workspace:*",
     "@fern-api/generator-commons": "workspace:*",
     "@fern-api/logging-execa": "workspace:*",
-    "@fern-fern/ir-sdk": "^32",
```

Comment:
didnt catch this in the review, but we shouldnt be depending on a specific version of IR in the template